### PR TITLE
Remove the account services url from the header

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?
Remove the account services url/"My Account" link from the header.

Amend the text on session expiry page so that it matches the design [here](https://ons-design-system.netlify.com/patterns/error-status-pages/#timeout).

### How to review 
Open any schema and check that the link no longer appears in the header.

Check to make sure that the text on the session expiry page has been amended to match the design above.

### CODE REVIEW

**Comment by berroar** [comment]
[Remove account_services_url from header](https://trello.com/c/SH4t1liW/3286-remove-accountservicesurl-from-header)

**Review by rmccar — APPROVED** [review]

File: templates/layouts/_base.html [hunk 1]
```diff
@@ -14,7 +14,6 @@
 } %}
 
 {% include 'layouts/configs/_languages.html' %}
-{% include 'layouts/configs/_service-links.html' %}
```
Comment (MebinAbraham) [inline templates/layouts/_base.html:17]: `_service-links.html` should be removed then?

**Review by MebinAbraham — COMMENTED** [review]
Not sure, but maybe `My Account` link should be configurable to display or not instead of completely removing?

File: templates/layouts/_base.html (hunk 1 above)
Comment (petechd) [reply templates/layouts/_base.html]: We left it because it'll be needed somewhere else in eq... We don't know where yet.

Comment (petechd) [reply templates/layouts/_base.html]: but if it's included it shows up in header element, when you give it url attribute which is the case.

Comment (petechd) [reply templates/layouts/_base.html:17]: It's been removed now, thanks.

File: package-lock.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
```
Comment (MebinAbraham) [inline package-lock.json]: Should be removed?

File: templates/errors/session-expired.html [hunk 3]
```diff
@@ -19,7 +19,9 @@ <h1 {{helpers.track('visible', 'Timeout', 'Timeout expired')}}>{{ _("Your sessio
   {% endcall %}
 
   {% if account_service_url %}
-    <a href="{{account_service_url}}">{{ _("Return to your account") }}</a>
+    <p>{{ _("We have saved your progress and you will need to ") }}
```
Comment (MebinAbraham) [inline templates/errors/session-expired.html]: This should help make the translation of this more meaningful. At the moment, they are split to 3 difference sentence which won't make sense?
```html
    <p>
    {{ _('We have saved your progress and you will need to %(enter_your_unique_code)s to access the survey again.', enter_your_unique_code=('<a href="' ~ account_service_url ~ '">%s</a>' % _('enter your unique code'))) }}
    </p>
```

Comment (berroar) [reply templates/errors/session-expired.html]: Updated this.

File: package-lock.json (hunk 2 above)
Comment (berroar) [reply package-lock.json]: Removed.

File: templates/errors/session-expired.html (hunk 3 above)
Comment (MebinAbraham) [reply templates/errors/session-expired.html]: As discussed:
```
    {{ _('We have saved your progress and you will need to <a href="%(account_service_link)s">enter your unique code</a> to access the survey again.', account_service_link=account_service_url) }}
```

**Review by JamesGardiner — APPROVED** [review]

**Pull request merged**